tensorflow_datasets/scripts/convert_huggingface.py

```diff
@@ -0,0 +1,87 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Add HuggingFace Datasets."""
```

Comment:
Please use absl.app, to run the script, like in https://github.com/tensorflow/datasets/blob/b9deaa07f567ad3211850a1e5c1ae993a1a03409/tensorflow_datasets/scripts/documentation/build_catalog.py#L149